### ISSUE TITLE
New rule to standardise style of aliased imports

### ISSUE THREAD

**Issue by ngnpope** [astral-sh/ruff#10608]
So, I've had a good look through all the available rules and found nothing that quite matches this.

I could have an aliased import in this style:

```python
import pyarrow.parquet as pq
```

Or in this style:

```python
from pyarrow import parquet as pq
```

It would be nice to select one form or the other. I imagine a setting would be a good idea as people have differing preferences. I think this should stay away from having per-package/module configuration to allow going both ways in different cases - it'll just over-complicate things.

This would probably fit nicely under the [`ICN`](https://docs.astral.sh/ruff/rules/#flake8-import-conventions-icn) set of rules?

FWIW, this is also quite close to [`PLR0402`](https://docs.astral.sh/ruff/rules/manual-from-import/), but that's only aimed at avoiding `import x.y as y` cases.
